web/packages/plugins/shared/common/src/components/Pickers/EntityPicker.tsx

```diff
@@ -1,94 +1,43 @@
-import React, { useContext, useState } from 'react'
+import React, { useState } from 'react'
 import { BlueprintEnum } from '../../utils/variables'
-import {
-  ApplicationContext,
-  DmssAPI,
-  Modal,
-  Spinner,
-  Tree,
-  TreeNode,
-  TReference,
-  useDataSources,
-} from '../../index'
-import { IDataSources } from '../../hooks/useDataSources'
-import { AuthContext } from '@dmt/common'
+import { Modal, TreeNode, TreeView, TReference } from '../../index'
 import { Input } from '@equinor/eds-core-react'
 
 export const EntityPicker = (props: {
   onChange: (ref: TReference) => void
   formData?: TReference
+  blueprintFilter?: string
 }) => {
   // TODO: Valid types should be passed to this, and filtered for in the view
-  const { onChange, formData } = props
+  const { onChange, formData, blueprintFilter = BlueprintEnum.PACKAGE } = props
```

Comment:
blueprintFilter can be removed from props and line 12

Reply:
done